cookiecutter.json

```diff
@@ -32,6 +32,7 @@
     "Gulp"
   ],
   "cloud_provider": [
+    "None",
```

Comment:
I think this will make it the default option, wouldn't it? We usually add new options at the end of the list.

Can you change this and the associated pieces of doc please?

Reply:
Sure, I'll change it